### PR TITLE
fix scan don't return result after add enable_multi_versions

### PR DESCRIPTION
**Pull request by critical27**
We just return all data if enable_multi_versions is false.

### CODE REVIEW

**Review by panda-sheep — APPROVED** [review]
Excellent!

**Comment by panda-sheep** [comment]
Is there documentation for the scan interface?
If so, it is recommended to follow the documentation.

If FLAGS_enable_multi_versions is false, the `start time` and `end time ` have no effect. 
Otherwise it will cause user confusion.

File: src/storage/query/ScanEdgeProcessor.cpp [hunk 1]
```diff
@@ -67,7 +68,7 @@ void ScanEdgeProcessor::process(const cpp2::ScanEdgeRequest& req) {
         // only return data within time range [start, end)
         EdgeVersion version = folly::Endian::big(NebulaKeyUtils::getVersion(key));
         int64_t ts = std::numeric_limits<int64_t>::max() - version;
-        if (ts < startTime || ts >= endTime) {
+        if (FLAGS_enable_multi_versions && (ts < startTime || ts >= endTime)) {
```
Comment (bright-starry-sky) [inline src/storage/query/ScanEdgeProcessor.cpp:71]: A question, if FLAGS_enable_multi_versions == false, here should be return all versions data? right?

Comment (critical27) [reply src/storage/query/ScanEdgeProcessor.cpp:71]: Yeah, we will return all versions. When `FLAGS_enable_multi_versions` is false, there should be only one version.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by darionyaphet — APPROVED** [review]
LGTM

**Comment by dangleptr** [comment]
Maybe we could skip the check logic inside compaction too.  Of course, we should open another pr for it.

**Pull request merged**